src/nevermined/utils/JwtUtils.ts

```diff
@@ -33,7 +33,7 @@ export class EthSignJWT extends SignJWT {
 
         let input = ethers.utils.arrayify(sign)
 
-        if (!isEtherSign) {
+        if (!isEtherSign && process.env.OLD_GATEWAY === 'true') {
```

Comment:
better leave a comment here to remind us to remove this after the migration is complete